app/views/rails_performance/layouts/rails_performance.html.erb

```diff
@@ -6,9 +6,6 @@
     <title>Rails Performance</title>
     <%= csrf_meta_tags %>
     <%= csp_meta_tag if ::Rails::VERSION::STRING.to_f >= 5.2 %>
-    <script>
-      window.APP_TIME_ZONE = '<%= Rails.application.config.time_zone.presence || 'UTC' %>';

```

Comment:
can we keep it and implement something like it?

Reply:
@vol1ura maybe some other options can be considered

Reply:
Here I try to use option `datetimeUTC: false` in charts options. It seems the time is displayed correctly now. Please, check

Reply:
hmm, maybe it works fine

could you please check if you have the same. I just see some inconsistency and don't know why

<img width="1863" alt="image" src="https://github.com/user-attachments/assets/d64ca927-6ff2-4be3-9a22-b333802e4ca1">

Reply:
also not sure about this part of the tooltip, I think it's obvious anyway

<img width="1302" alt="image" src="https://github.com/user-attachments/assets/1c0461f3-e382-4a85-a922-1c1389675fb3">


also not sure but I have a feeling this chart library works slower on mouse hover

but I like the tooltip with time on X-axis

Reply:
another example of chart library that can be used: https://echarts.apache.org/examples/en/index.html

I used it few times and it's quite powerful

Reply:
> also not sure about this part of the tooltip, I think it's obvious anyway

Tooltip title can be disabled on all charts, I'll add this param

Reply:
@vol1ura please also move Y axis to the left side. Basically, make a chart very similar to the current version.

After that, I think I can try to release "alpha" version and try it on production